### DISCUSSION TITLE
Mute automation for a chosen time

### DISCUSSION THREAD

**Discussion by Fifke** [home-assistant/frontend#16482] [Automations / scripts / scenes]
I would love to see the ability to mute an automation during a period... Instead of only turn it off. For example the automation i have on my bedroom echo when someone rings the doorbel. I shut it dowm while our todler takes a nap... Also automations regarding the heating thermostat are switched off during summer... Would be nice to be alble to choose a period for it to be switched off , from just minutes to months...

**Comment by pimw1** [comment]
You can achieve this by using the automation.turn_off service. You can put this in an automation and let it be triggered by calendar events, time, an input boolean representing the todler taking a nap, etc.

**Comment by pimw1** [comment]
A more elegant approach is to add an input boolean to your automation as a condition.  You can use all the triggers you can imagine to turn on/off that input boolean.
